mplhep/alice.py

```diff
@@ -0,0 +1,43 @@
+# Log styles
+from . import styles_alice as style
+from . import label as label_base
+from .label import lumitext
+import mplhep._deprecate as deprecate
+from matplotlib import docstring
+
+
+__all__ = [style, lumitext]
+
+
+# Experiment wrappers, full names made private
+def _alice_text(text="", **kwargs):
+    return label_base._exp_text(
+        "ALICE", text=text, fontsize=28, loc=1, italic=(False, False), **kwargs
+    )
+
+
+def _alice_label(**kwargs):
+    return label_base._exp_label(
+        exp="ALICE", fontsize=28, loc=1, italic=(False, False), rlabel="", **kwargs
+    )
+
+
+# Temporary aliases
+@deprecate.deprecate("Naming convention is changing. Use ``mplhep.alice.label``.")
+def alicelabel(*args, **kwargs):
+    return _alice_label(*args, **kwargs)
+
+
+@deprecate.deprecate("Naming convention is changing. Use ``mplhep.alice.text``.")
+def alicetext(**kwargs):
+    return _alice_text(**kwargs)
+
+
```

Comment:
You can remove these, no need to add new stuff in an already deprecated mode.

Reply:
Done :)